MovieApp/MovieApp/Presenters/HomePagePresenter.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+final class HomePagePresenter {
+    
+    var movies: [Movie] {
+        let movies = [
```

Comment:
Since this is a mock it doesn't matter, but Swift allows you to omit `return` so you could've just written:
```swift
var movies: [Movie] {
    [
        Movie(
            title: "Iron Man (2008)",
            desription: "After being held captive in an Afghan cave, billionaire engineer Tony Stark creates a unique weaponized suit of armor to fight evil.",
            imageString: "MockupMovieImage.pdf"),
        Movie(
            ...
    ]
}
```

Also, please check the style guide on how to write multi-line function calls (specifically where does the first argument goes).